.github/workflows/consuming.yml

```diff
@@ -18,17 +18,32 @@ jobs:
       matrix:
         project: ['admiral', 'submariner', 'submariner-operator', 'lighthouse']
         deploytool: ['operator', 'helm']
+        cabledriver: ['libreswan']
         exclude:
           # Admiral E2E doesn't respect deploy-tool params, as it uses clusters without Submariner
           - project: admiral
             deploytool: helm
           # Operator and Helm are mutually exclusive, don't try to use Helm in Operator repo
           - project: submariner-operator
             deploytool: helm
+        include:
+          # Parallel the consuming projects' test matrices for cable driver CI coverage
```

Comment:
I'm not sure what this means..
From the name of the PR I thought it's about running in parallel?

Perhaps something like:
"Test additional combinations of cable drivers on select consuming projects"

Reply:
It means we test the same combinations here as the consuming projects do in their CI. They aren't just additional, they make the two sets of tests equal. I can try to reword it to be more clear.